x/bank/spec/05_params.md

```diff
@@ -6,7 +6,20 @@ order: 5
 
 The bank module contains the following parameters:
 
-| Key         | Type | Example |
-|-------------|------|---------|
-| sendenabled | bool | true    |
+| Key                | Type          | Example                            |
+|--------------------|---------------|------------------------------------|
+| SendEnabled        | []SendEnabled | [{denom: "stake", enabled: true }] |
+| DefaultSendEnabled | bool          | true                               |
 
+
+## SendEnabled
+
+The send enabled parameter is an array of SendEnabled entries mapping coin
+denominations to their send_enabled status.  Entries in this list take
+precedence over the `DefaultSendEnabled` setting.
+
+## DefaultSendEnabled
```

Comment:
I'm not really convinced this is necessary or provides a cleaner UX. What is the drawback to just using `SendEnabled`? I can see people getting confused over this IMHO.

Reply:
I agree. I think handling this at the client level would be more appropriate